ffs/scheduler/scheduler.go

```diff
@@ -235,9 +240,10 @@ func (s *Scheduler) run() {
 			log.Infof("scheduler daemon terminated")
 			return
 		case <-s.sd.evaluateQueue:
-			log.Debug("evaluating storage job queue...")
+			log.Infof("storage job push rate limit: %d/%d", len(s.sd.rateLim), cap(s.sd.rateLim))
+			stats := s.sjs.GetStats()
+			log.Infof("storage job total queued: %d, total executing: %d", stats.TotalQueued, stats.TotalExecuting)
```

Comment:
Every time the job queue is evaluated to see if any queued job is runnable, we print some extra information like:
- How close we're to the rate limit
- queued jobs
- executing jobs

I preferred to be more verbose to include these things in logs to understand what's happening.